c/index.md

```diff
@@ -0,0 +1,10 @@
+# List of the artists starting with C
```

Comment:
What is the title meant to say?

Reply:
Shaun,could please check again, we've made changes.